scripts/test-fuzzed-padding.py

```diff
@@ -149,35 +191,48 @@ def main():
     # run the conversation
     good = 0
     bad = 0
-
-    for c_name, c_test in conversations.items():
+    failed = []
+    if not num_limit:
```

Comment:
This is true when `num_limit` is set to zero. maybe better to write `if num_limit is not None`?